catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -311,7 +328,36 @@ public AttributeFormat getAttributeFormat() {
                 false /* tokenized */,
                 true /* multivalued */,
                 STRING_TYPE));
-        return descriptors;
+        DESCRIPTORS = Collections.unmodifiableSet(descriptors);
+
+        BASIC_METACARD = new MetacardTypeImpl(MetacardType.DEFAULT_METACARD_TYPE_NAME,
```

Comment:
I don't agree that `new FooBar()` is any uglier than `FooBar.STATIC_FOO_BAR`. Besides, this a much more OO design that hides and encapsulates the implementation detail of where these attributes are populated from. Exposing the Type as a static object is an old design pattern that I'm no longer convinced is worth perpetuating.